config/printer-biqu-b1-se-plus.cfg

```diff
@@ -0,0 +1,219 @@
+endstop_pin: probe:z_virtual_endstop
+homing_speed: 10
+second_homing_speed: 1
+position_min: -2
+position_max: 340
+
+[tmc2208 stepper_z]
+uart_pin: PD0
+run_current: 0.800
+hold_current: 0.400
+stealthchop_threshold: 999999
+
+########################################
+# Extruder Pins and TMC2208 configuration
+########################################
+[extruder]
+step_pin: PD15
+dir_pin: !PD14
+enable_pin: !PC7
+microsteps: 16
+rotation_distance: 34.2152 # Calibrar - ver https://www.klipper3d.org/Rotation_Distance.html
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PB3
+sensor_type: Generic 3950
+sensor_pin: PA2 #thermistor pin
+control: pid
+pid_Kp: 22.2
+pid_Ki: 1.08
+pid_Kd: 114
+min_temp: 0
+max_temp: 250
+
+[tmc2208 extruder]
+uart_pin: PC6
+run_current: 0.800
+hold_current: 0.300
+stealthchop_threshold: 999999
+########################################
+
+########################################
+# Heater Bed Pins
+########################################
+[heater_bed]
+heater_pin: PD7
+sensor_type: Generic 3950
+sensor_pin: PA1
+control: pid
+pid_Kp: 54
+pid_Ki: 0.77
+pid_Kd: 900
+min_temp: 0
+max_temp: 110
+
+########################################
+# Printer Configuration
+########################################
+[printer]
+kinematics: cartesian
+max_velocity: 200
+max_accel: 1000
+max_z_velocity: 5
+max_z_accel: 100
+
+########################################
+# Probe configuration
+########################################
+[probe]
+pin: ^!PE4
+z_offset: 0.0
+x_offset: 0.0
+y_offset: 0.0
+speed: 10.0
+samples: 2
+samples_result: average #median
+sample_retract_dist: 2.0
+samples_tolerance: 0.2
+
+[safe_z_home]
+home_xy_position: 155,155   #140,169
+speed: 100
+z_hop: 5
+z_hop_speed: 5
+
+[output_pin probe_enable]
+pin: PE5
+value: 1
+
+########################################
+# Bed Mesh configuration               # exemple off bed mesh config
+########################################
+#[bed_mesh]
+#speed: 2000
+#horizontal_move_z: 3
+#mesh_min: 20, 20
+#mesh_max: 290, 290
+#probe_count: 7, 7                  # 49 points due to large bed size
+#mesh_pps: 2,2
+#algorithm: bicubic
+#bicubic_tension: 0.2
+#fade_start: 1
+#fade_end: 3
+#fade_target: 0
+
+########################################
+# Fan Nozzle configuration
+########################################
+[fan]
+pin: PB7
+
+[heater_fan Cooling_fan]
+pin: PB6
+max_power: 1.0
+kick_start_time: 0.100
+heater: heater_bed
+heater_temp: 40.0
+fan_speed: 1.0
+
+[heater_fan Board_fan]
+pin: PB5
+max_power: 1.0
+kick_start_time: 0.100
+heater: extruder
+heater_temp: 40.0
+fan_speed: 1.0
+########################################
+
+########################################
+# Filament Sensor configuration
+########################################
+[filament_switch_sensor Sensor_Filamento]
+switch_pin: !PC2
+pause_on_runout: false #pause handled by macro
+runout_gcode:
+  M600 #calls a macro with the name M600 (so it can be called by slicers too)
+insert_gcode:
```

Comment:
You can remove optional features like insert_gcode.  Having it call a throwaway gcode will end up being an error a user may have to try and figure out

Reply:
On this point you are right.  I will delete as soon as I see it